x-pack/plugins/triggers_actions_ui/public/application/sections/test_utils.tsx

```diff
@@ -52,13 +52,13 @@ export const createAppMockRenderer = (
 
   const AppWrapper: React.FC<{ children: React.ReactElement }> = React.memo(({ children }) => (
     <I18nProvider>
```

Comment:
Why is `I18nProvider` needed in these test utilities but not in `x-pack/plugins/alerting/public/lib/test_utils.tsx`? 🤔

Reply:
Thanks to @clintandrewhall for helping me understand the reason. The ideal structure of the code would be to not include `<I18nProvider>` in AppWrapper. However, KibanaRenderContextProvider only provides a _mock_ of `core.i18n`, and the particular mock does not add the context that would work. Therefore, the extra boilerplate of `<I18nProvider>` wrapper is still needed.

Filed an issue to be able to bring this kind of test code to the ideal state: https://github.com/elastic/kibana/issues/180725